components/prism-sas.js

```diff
@@ -130,6 +193,43 @@
 			}
 		},
 
+		'proc-cas': {
+			pattern: /(^proc\s+cas(?:\s+[\w|=]+)?;)[\s\S]+?(?=^(?:proc\s+\w+|quit|data);|(?![\s\S]))/im,
+			lookbehind: true,
+			inside: {
+				'statement-var': {
+					pattern: /((?:^|\s)=?)(?:saveresult)(?:\s)+(?:[\s\S]+?)(?=;)/im,
+					lookbehind: true,
+					inside: {
+						'statement': {
+							pattern: /^(?:saveresult)(?:\s+[\S]+?)(?=\s)/i,
```

Comment:
1. `[\S]` == `\S`
2. `\S+?(?=\s)` == `\S+`

```suggestion
							pattern: /^saveresult\s+\S+/i,
```

Reply:
Incorporated in commit.